encryption_works.md

```diff
@@ -317,11 +317,7 @@ For a full explanation of how it works and how to set it up, check out the Inter
 
 Like PGP, OTR is used for two things: **encrypting the contents** of real-time instant message conversations and **verifying the identity** of people you chat with. Identity verification is important, and something many OTR users neglect to do. While OTR is more user-friendly than other types of encryption, there are some things you should know about OTR to understand it fully and know what attacks against it are possible.
 
-OTR encrypts the contents of your chats but not the metadata related to them -- who you talk to, how often, and when. You can use Google Talk and Facebook Chat with OTR, but we know that these services comply with surveillance requests from many governments that leave your metadata exposed, even if you're encrypting the content of your IM conversations. This is another reason to use XMPP, since it supports sending data over the Tor network, adding an additional layer of protection from metadata analysis.
-
-For this reason, Freedom of the Press Foundation recommends using a chat service that doesn't collaborate with intelligence agencies, such as Jabber. It's not a guarantee against metadata gathering, but it's better than nothing. OTR works with Google Talk and Facebook Chat, but Freedom of the Press Foundation recommend using XMPP, also known as Jabber. Jabber is an open protocol, and, unlike Facebook and Google, makes it harder for conversation metadata to be found. This is good for journalists who want to speak more securely to whistleblowers.
-
-Jabber servers you can get free accounts with include [Calyx Institute](https://www.calyxinstitute.org/projects/public_jabber_xmpp_server), [Riseup](https://help.riseup.net/en/chat) and [Duck Duck Go](https://duck.co/blog/post/4/xmpp-services-at-duckduckgo).
+OTR encrypts the contents of your chats but not the metadata related to them -- who you talk to, how often, and when. You can use Google Talk and Facebook Chat with OTR, but we know that these services comply with surveillance requests from many governments that leave your metadata exposed, even if you're encrypting the content of your IM conversations. For this reason, Freedom of the Press Foundation recommends using XMPP services that demonstrably take the necessary precautions to ensure users' privacy on all fronts: XMPP servers that support communication over Tor, suppress metadata between servers, enable OTR at the very beginning of a conversation, and have privacy-friendly stances towards logging.   XMPP servers hosted by [Calyx Institute](https://www.calyxinstitute.org/projects/public_jabber_xmpp_server), [Riseup](https://help.riseup.net/en/chat) and [Duck Duck Go](https://duck.co/blog/post/4/xmpp-services-at-duckduckgo) score really highly in this regard.
```

Comment:
1. I really like how you re-worded the "Freedom of the Press Foundation recommends" sentence.
2. What does "suppress metadata between servers" mean? I don't know enough about XMPP to understand how this might be accomplished. Is this just referring to encrypting server <-> server communication? (If so, how does that actually work? Seems like it would have a lot of the same problems as email...)
3. "Enable OTR at the beginning of a conversation" - no servers currently do that (the server can't initiate OTR for you). Calyx blocks all messages that are not OTR-encrypted, which forces you to enable OTR in order to converse, which has the same ultimate effect but is a bit different. I would say "require OTR encryption" instead.
4. "privacy-friendly logging policies" instead of "privacy-friendly stances towards logging"?